qemu/tests/expose_host_mtu.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+from avocado.utils import process
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_test
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Expose host MTU to guest test
+
+    1) Boot up guest with param 'host_mtu=4000' in nic part
+    2) Disable NetworkManager in guest
+    3) set mtu of guest tap (eg: tap0) and physical nic (eg: eno1) to 4000 in host
+    4) check the mtu in guest
+    5) ping from guest to external host with packet size 3972
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    host_interface = None
+    if params["netdst"]:
+        host_interface = params["netdst"]
```

Comment:
If I understand it correctly, here `netdst` is the device name, if other user does not name their bridge as `switch` or `ovs0`, this case will fail. Do we have more solid method to judge if the bridge is ovs or switch? I suppose we should be able to know it when we get the device name from `params["netdst"]`. Please correct me if I am wrong. Thanks.

Reply:
Yes we could get the device name from netdst. But we cannot make sure how the user name their device. For eg: some may name ovs bridge as switch. I cannot find out a better way to check the backend. Any suggestions will be welcomed.

Reply:
Or I got a better idea to check whether it's an ovs bridge by: 
    if netdst not in process.system_output("ovs-vsctl list-br").decode():
        test.error("%s isn't an openvswith bridge" % netdst)

Reply:
@xiywang Can we assume it is switch if not get netdst in "ovs-vsctl list-br", since for sure the nettype is bridge, it would be either ovs or switch.

Reply:
@xiywang Similar here: https://github.com/autotest/tp-qemu/blob/master/generic/tests/netperf.py#L134

Reply:
Juse like the occasion in netperf.py, there're three backends: ovs, bridge and macvtap. Guess we can follow the usage in that file.

Reply:
@vivianQizhu Updated. Please help review again. Thanks.